tools/SetupFlow/DevHome.SetupFlow/ViewModels/MainPageViewModel.cs

```diff
@@ -30,7 +30,7 @@ namespace DevHome.SetupFlow.ViewModels;
 /// </summary>
 public partial class MainPageViewModel : SetupPageViewModelBase
 {
-    private readonly ILogger _log = Log.ForContext("SourceContext", nameof(EnvironmentsSetupFlowFeatureName));
+    private readonly ILogger _log = Log.ForContext("SourceContext", nameof(MainPageViewModel));

```

Comment:
Not the fault of this PR, but we have two classes in Dev Home named MainPageViewModel, one in SetupFlow and one in Configuration. @dkbennett Will this cause confusion in the logs? Should we be including more of the namespace in those files to differentiate them? (cc: @AmelBawa-msft @nieubank)